tests/test_core.py

```diff
@@ -4105,6 +4105,12 @@ def test_dylink_hyper_dupe(self):
       full = run_js('src.cpp.o.js', engine=JS_ENGINES[0], full_output=True, stderr=STDOUT)
       self.assertContained("warning: trying to dynamically load symbol '_sideg' (from 'third%s') that already exists" % dylib_suffix, full)
 
+  @needs_dlfcn
+  def test_dylink_dso_needed(self):
+      def do_run(src, expected_output):
+          self.do_run(src + 'int main() { return _main(); }', expected_output)
+      self._test_dylink_dso_needed(do_run)
```

Comment:
2 space indentation

Reply:
Sorry, I usually do `set ts=8 sts=2 sw=2 et` in vim when working on emscripten, but missed this again. Will fix. (btw, a way to automatically check/reformat this would be helpful)

Reply:
We use flake8 which should check for indentation but sadly its set of 8 spaces and is not configurable we so had to disable that test completely.

If we could switch to 4 spaces, or find a better tool I'd be happy.   Switching at this point would be pretty invasive though.

Reply:
I see.